src/components/MenuItem.tsx

```diff
@@ -363,6 +367,7 @@ function MenuItem(
         [
             styles.flexShrink1,
             styles.popoverMenuText,
+            shouldCenter ? styles.textAlignCenter : {},
```

Comment:
Here we used an object and other places let `false` exist? Maybe we can be more consistent about this.

Reply:
Yeah, I tend to agree that an empty object is clearer though would prefer it not be there at all. I think I'm more concerned about consistency here. Whatever we do, let's do it the same way everywhere.